.rubocop.yml

```diff
@@ -3,6 +3,8 @@ AllCops:
     - mixlib-install.gemspec
 Style/AlignParameters:
   EnforcedStyle: with_first_parameter
+Style/BlockDelimiters:
```

Comment:
I think its the right thing but unrelated to the change in this PR. I hit this as well in #134 and have submitted #136 to fix this separately.